### PR TITLE
Crit people can cry

### PR DESCRIPTION
**Pull request by KittyNoodle**
## About The Pull Request
You can now cry while in crit. I honestly thought this was a thing before and was surprised it wasn’t.

## Why It's Good For The Game
This has no mechanical advantage and is just flavor. Let’s you tell the desword/revolver/whatever is the meta murderbone weapon user how you really feel as they kill you.

## Changelog
:cl:
add: You can now cry when in crit
/:cl:

### CODE REVIEW

**Review by Fikou — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -10,6 +10,7 @@
 	message = "cries."
 	message_mime = "sobs silently."
 	emote_type = EMOTE_AUDIBLE | EMOTE_VISIBLE
+	stat_allowed = HARD_CRIT
```
[the remark this answers is not in the document]
Comment (KittyNoodle) [reply code/modules/mob/living/carbon/human/emote.dm:13]: You could gasp before while unconscious but I will try and fix this

[the remark this answers is not in the document]
Comment (Fikou) [reply code/modules/mob/living/carbon/human/emote.dm:13]: yes because youre dying of no oxygen

**Comment by Overuseddart** [comment]
Emote Powercreep! This will lead to stun gloves being merged back into the game this is way too strong. (Finally I can cry after losing in the rage cage)

**Comment by distributivgesetz** [comment]
time to powergame for empathy points with this pr

File: code/modules/mob/living/carbon/human/emote.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (KittyNoodle) [reply code/modules/mob/living/carbon/human/emote.dm]: fixed it

**Comment by Xander3359** [comment]
Speedmerge this, people who kill me should need to see me cry.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -10,6 +10,7 @@
 	message = "cries."
 	message_mime = "sobs silently."
 	emote_type = EMOTE_AUDIBLE | EMOTE_VISIBLE
+	stat_allowed = SOFT_CRIT
```
Comment (Mothblocks) [inline code/modules/mob/living/carbon/human/emote.dm:13]: Does this still let you do it when you're conscious?

Comment (KittyNoodle) [reply code/modules/mob/living/carbon/human/emote.dm:13]: yep, works perfectly 
when unconscious in crit:
image

when conscious in crit:
image

**Comment by Shadowflame909** [comment]
true ss13 empathy restored

**Review by Mothblocks — APPROVED** [review]

**Comment by Farquaar** [comment]
I think it should only work in softcrit, personally.

**Pull request merged**